### PR TITLE
Add a github actions doc build

### PR DESCRIPTION
**Pull request by dstansby**
Still to do:

- [x] Replace pyvista example with sunkit-pyvista example
- [ ] Have examples render output
- [x] Push the rendered docs to a github pages

Url is https://sunpy.github.io/sunkit-pyvista/

### CODE REVIEW

File: .github/workflows/build_docs.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+name: Build and Deploy Docs
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
```
Comment (nabobalis) [inline .github/workflows/build_docs.yml:6]: To be removed before merge.

File: sunkit_pyvista/plotter.py [hunk 2]
```diff
@@ -169,7 +170,7 @@ def plot_map(self, m, clip_interval: u.percent = None, **kwargs):
                                  "specified as two numbers.")
         else:
             clim = [0, 1]
-        self.plotter.add_mesh(map_mesh, cmap=cmap, clim=clim, **kwargs)
+        self.plotter.add_mesh(map_mesh, cmap="RdYlBu", clim=clim, **kwargs)
```
Comment (nabobalis) [inline sunkit_pyvista/plotter.py:173]: To be reverted.

**Comment by dstansby** [comment]
Right, I've narrowed the issue down to `ipygany` not showing meshes that have some `nan` vertices in them. This is common for images of the Sun, as all the off-disc coordinates are set to `nan`.

**Comment by dstansby** [comment]
As a bit more of a paper trail, here's the relevant issue (or possibly it's a feature request): https://github.com/QuantStack/ipygany/issues/63

**Pull request closed without merging**